scripts/scr_apothecary_ship/scr_apothecary_ship.gml

```diff
@@ -68,36 +68,14 @@ function scr_apothecary_ship() {
 	// show_message("[1] "+string(obj_ini.ship[1])+": "+string(ship_apoth[1])+" apothecaries");
 
 
-	var normal_hp,mixhp,ratio;
+	var normal_hp,mixhp,ratio,unit, unit_location;
```

Comment:
`unit` is declared twice, once here, second line 105. I suggest to just remove this one.
`unit_location` is not used at all